packages/helm-charts/oracle/USDCUSD.yaml

```diff
@@ -3,8 +3,8 @@ oracle:
   aggregation:
     mid:
       maxExchangeVolumeShare: 1
-      maxPercentageDeviation: 0.05
-      maxPercentageBidAskSpread: 0.025
+      maxPercentageDeviation: 0.01
+      maxPercentageBidAskSpread: 0.005
```

Comment:
how did you arrive at 0.5% here?

Reply:
I think for most stables market makers will try to keep a b/a spread sub 10bps. In that sense if the b/a spread is at 50bps I guess either the market maker is offline or there is a lot of uncertainty in the market

Reply:
ok thanks